frontend/src/components/AdminControls/Users.js

```diff
@@ -1,133 +1,216 @@
+        headerStyle: {
+          textAlign: 'center',
+          borderTop: 'none',
+          borderBottom: 'none',
+        },
+        formatter: (cell) => {
+          let dateObj = cell;
+          if (typeof cell !== 'object') {
+            dateObj = new Date(cell);
+          }
+
+          const day = `0${dateObj.getUTCDate()}`.slice(-2);
+          const month = `0${dateObj.getUTCMonth() + 1}`;
+          const year = dateObj.getUTCFullYear();
+          return `${month}/${day}/${year}`;
+        },
+        editor: {
+          type: Type.DATE,
+        },
       },
-      formatter: cell => `${cell.length}`,
-    },
-    {
-      dataField: 'edit',
-      text: '',
-      headerStyle: {
-        textAlign: 'center',
-        borderTop: 'none',
-        borderBottom: 'none',
+      {
+        dataField: 'recipes',
+        text: 'Number Of Recipes',
+        align: 'center',
+        style: {
+          paddingRight: '80px',
+        },
+        headerStyle: {
+          paddingRight: '80px',
+          textAlign: 'center',
+          borderTop: 'none',
+          borderBottom: 'none',
+        },
+        formatter: cell => `${cell.length}`,
       },
-      align: 'left',
-      formatter: () => (
-        <div>
-          <button type="button" className="admin-edit-button">
-            <img
-              src="https://img.icons8.com/windows/32/000000/edit.png"
-              alt="edit"
-            />
-          </button>
-          <button type="button" className="admin-edit-button">
-            <img
-              src="https://img.icons8.com/windows/32/000000/cancel.png"
-              alt="delete"
-            />
-          </button>
-        </div>
-      ),
-    },
-  ];
-
-  const options = {
-    hideSizePerPage: true,
-    sizePerPageList: [
       {
-        text: '5',
-        value: 20,
+        dataField: 'edit',
+        text: '',
+        headerStyle: {
+          textAlign: 'center',
+          borderTop: 'none',
+          borderBottom: 'none',
+        },
+        align: 'left',
+        formatter: () => (
+          <div>
+            <button type="button" className="admin-edit-button" onClick={this.toggleEditModal}>
+              <img
+                src="https://img.icons8.com/windows/32/000000/edit.png"
+                alt="edit"
+              />
+            </button>
+          </div>
+        ),
       },
+      // Separate column added for suspend-user icons.
+      // Will display one icon or the other based on whether
+      // user is suspended or not.
       {
-        text: 'All',
-        value: users.length,
+        dataField: 'is_suspended',
+        text: '',
+        headerStyle: {
+          textAlign: 'center',
+          borderTop: 'none',
+          borderBottom: 'none',
+        },
+        align: 'left',
+        formatter: (cell) => {
+          if (cell) {
+            return (
+              <div>
+                <button type="button" className="admin-edit-button">
```

Comment:
This one and the one below should probably have a different className (something like admin-suspend-button?) 
Then you could also fix the sizing because I think currently when a user is suspended, the icon shows up as 25x25 and looks a bit small (the others are 32x32)